ecomap/src/python/ecomap/config.py

```diff
@@ -0,0 +1,66 @@
+from ConfigParser import SafeConfigParser
+import logging
+import time
+import unittest
+
+logger = logging.getLogger('example')
+logging.basicConfig(level=logging.DEBUG)
+REFRESH_TIME = 15 * 60                           # 15 minutes
+
+
+class Config(object):
+
```

Comment:
Could you please add docstrings to class, methods:
Example:
"""Replaces template placeholder with values.

:param timestamp: formatted date to display
:param priority: priority number
:param priority_name: priority name
:param message: message to display
:returns: formatted string
"""